language/move-vm/runtime/src/runtime.rs

```diff
@@ -90,7 +90,7 @@ impl VMRuntime {
         script: Vec<u8>,
         ty_args: Vec<TypeTag>,
         mut args: Vec<Value>,
-        sender: AccountAddress,
+        signers: Vec<Value>,
```

Comment:
this could be a Vector<AccountAddress> right?

Reply:
yes! Updated the PR to make this change.